### ISSUE TITLE
svg do not have proper accessibility attribute

### ISSUE THREAD

**Issue by allen138** [SAP/ui5-webcomponents#1581]
**Describe the bug**
When running an axe accessibility test, elements with an svg fail because they are missing a title, aria-label, or aria-labelledby attribute. It only needs one of the aforementioned items in order to pass the axe test.

**To reproduce**
1. Go to 'https://sap.github.io/ui5-webcomponents/playground/components/Icon/'
2. Add the axe chrome extension
3. Open the dev tools and analyze the axe report
4. Notice the svg error

**Expected behavior**
svg elements will pass the axe test

**Screenshots**
![ui5-icon AXE test failure](https://user-images.githubusercontent.com/38701667/81085650-56ccc180-8eac-11ea-9787-8247cd323807.PNG)

![sample axe svg failure](https://user-images.githubusercontent.com/38701667/81087052-1a01ca00-8eae-11ea-907c-d68f86f8bccf.PNG)

**Context**
- UI5 Web Components version 
- OS/Platform: any
- Browser *(if relevant)*: all

**Affected components**
ui5-icon

**Comment by vladitasev** [comment]
Hello @allen138 

`ui5-icon` does have ACC support, in the form of the `accessibleName` property, respectively `accessible-name` attribute.

For example:
```html
<ui5-icon accessible-name="Add more options" name="add-equipment"></ui5-icon>
```

The markup that is generated is `aria-label` on the `<svg>` element:
```
aria-label="{{accessibleNameText}}"
```

This is how `accessible-name` works:
 - If given by the user, it is going to be used
 - If not present, check if the icon has a predefined default accessible name. If yes, use this. *Note*: very few icons have such.
 - Otherwise, no accessible name will be used.

In essence, the **user** is supposed to provide a semantically correct accessible name for each icon they use. For very few selected icons, translated predefined names exist, but they are only the ones used for `valueState`, so that each component that has `valueState` (all of the input ones) does not have to do this again and again:

```
#XTXT
ICON_MESSAGE_ERROR=Error

#XTXT
ICON_MESSAGE_INFORMATION=Information

#XTXT
ICON_MESSAGE_SUCCESS=Successful

#XTXT
ICON_MESSAGE_WARNING=Warning
```

In summary: we provide hundreds of icons which can't have universal enough `aria-label`s to make them really useful, so we leave that to the application. For icons that we use internally in our components, hence inaccessible to the application, we either provide built-in accessible names (for the value-state-related icons) or we provide accessibility otherwise, most commonly `aria-label` on the HTML element, holding the icon - be it search field, expand/collapse button, etc... so it's not necessary to have acc info on the icon itself, as it would duplicate what's on the higher-level element.

Hope that answers your questions. Please let us know if you need more information.

Regards,
Vladislav

**Comment by allen138** [comment]
Hi @vladitasev,
Thanks for the update. This will fix the majority of our issues. But how about the scenario where the icon is inside a button? We don't necessarily want to give the svg an aria-label, but instead possibly change the role from 'img' to 'presentation'. Is it possibly to manipulate the role of the icon's svg?

**Comment by stoehr** [comment]
> Hi @vladitasev,
> Thanks for the update. This will fix the majority of our issues. But how about the scenario where the icon is inside a button? We don't necessarily want to give the svg an aria-label, but instead possibly change the role from 'img' to 'presentation'. Is it possible to manipulate the role of the svg?

Another way to say this is as follows:  What if an icon is used for presentation only?  Can there be a component attribute that lets us tell the icon that it is used only for presentation?  And then under the covers, you guys can do whatever is needed to allow the underlying native HTML to signify this?  (So for the current implementation of the icon, it would be the underlying svg tag's "role" being changed to "presentation" that would accomplish this.)
